pkg/logproto/logproto.proto

```diff
@@ -24,11 +24,15 @@ message PushResponse {
 
 message QueryRequest {
   string query = 1;
-  uint32 limit = 2;
```

Comment:
I see what you've done here trying to reduce repetition but I would prefer to keep the query request flat and repeat what is needed in the tailing request.

Reply:
Hrm, this allowed reducing repetition in the actual code logic as well.. e.g. parsing http params into Lookback is required for both, query and tail

Reply:
I agree with @Kuqd I think I prefer the flat query request as well.

Also in general we would want to avoid breaking changes to a protobuf definition, adding new fields is ok, moving/renaming old fields should be avoided and instead they can be deprecated.

I also think rather than a lookback object I would prefer `lookbackStart` and `lookbackLimit` (do we need end?? wouldn't it always just end at the time the tail command was issued?)

Reply:
Tailing yes, but when Querying it must be possible to get old values. If you look at `http.go`, end is always forced to the current time for tailing.

So yes, we only need `start` and `limit` for tailing

Reply:
Okay, flattened these into the queries